sessionctx/binloginfo/binloginfo.go

```diff
@@ -14,6 +14,7 @@
 package binloginfo
 
 import (
+	"github.com/pingcap/parser/model"
```

Comment:
Please put this line in suitable location.

Reply:
Roger that!